### ISSUE TITLE
How to show new files?

### ISSUE THREAD

**Issue by ewauq** [rtfpessoa/diff2html#330]
Hello, 

I know that the question has already been asked but I don't understand answers. I am sorry if the answer is really simple but I can't get it...

I am using `diff2html -u pbcopy` to get all diffs in my git repo. In vscode I see all new/updated/deleted files, but when using diff2html I only get updated files and not new ones.

What is the command or option to use to get **updated and new files** together on diffy.org?

Thank you.

**Comment by rtfpessoa** [comment]
@ewauq diff2html will wrap a command to git so what you can do to have all the control is for example: `git diff HEAD | diff2html -i stdin -u pbcopy`

Does this solve your problem?

**Comment by ewauq** [comment]
@rtfpessoa Thank you for the answer!

For some reason the command you provided did not work for me. I am not a git expert, but I searched and found this  [stackoverflow thread](https://stackoverflow.com/questions/855767/can-i-use-git-diff-on-untracked-files). I just needed to do another git command before to do yours:

```bash
git add -N .
git diff HEAD | diff2html -i stdin -u pbcopy
```

Everything is fine now! I think it would very usefull to put this somewhere in the doc or in the FAQ.

Thank you!
